src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -4894,6 +4894,41 @@ namespace ThenIncludeIntellisenseBug
             End Using
         End Function
 
+        <MemberData(NameOf(AllCompletionImplementations))>
+        <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TestCompletionInsideMethodsWithDelegatesAsArguments(completionImplementation As CompletionImplementation) As Task
```

Comment:
Why is this a command handler test versus one in https://github.com/dotnet/roslyn/blob/master/src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs? (I recognize the other tests got added here, but the same question should have been applied to them as well.)